Source/WebKit/NetworkProcess/NetworkDataTask.h

```diff
@@ -176,10 +180,22 @@ class NetworkDataTask : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<N
     String m_pendingDownloadLocation;
     WebCore::ResourceRequest m_firstRequest;
     WebCore::ResourceRequest m_previousRequest;
-    bool m_shouldClearReferrerOnHTTPSToHTTPRedirect { true };
     String m_suggestedFilename;
+    size_t m_totalBytesTransferredOverNetwork { 0 };
+    size_t m_bytesTransferredOverNetworkReported { 0 };
+    bool m_shouldClearReferrerOnHTTPSToHTTPRedirect { true };
     bool m_dataTaskIsForMainFrameNavigation { false };
     bool m_failureScheduled { false };
 };
 
+size_t NetworkDataTask::bytesTransferredOverNetworkDelta()
```

Comment:
I don't think this needs to be inline. We should probably just move it to the cpp file to avoid polluting the header.

Reply:
But then in `NetworkLoad::bytesTransferredOverNetworkDelta() const` we need to ref `m_task`.

Reply:
I don't understand what your comment has to do with making the function inline.